src/rules/ResourceDNTTerms.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * ResourceDNTTerms.js - rule to ensure that Do Not Translate terms have not been translated
+ *
+ * Copyright © 2023 JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Rule, Result } from "i18nlint-common";
+import fs from "node:fs";
+
+/** Rule to ensure that Do Not Translate terms have not been translated;
+ * i.e., if a DNT term appears in source, it has to appear in target as well */
+class ResourceDNTTerms extends Rule {
+    /** @readonly */ name = "resource-dnt-terms";
```

Comment:
To make it readonly, use `#name = "resource-dnt-terms";`

Reply:
Actually, `#` makes it private https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields.

To make it truly readonly, one could probably use `Object.defineProperty` with `writeable: false`, but I feel like it does not matter _that much_ here to warrant the boilerplate - I just put the JSDoc annotation for the IntelliSense/completion purposes mostly